airbyte-integrations/bases/base-typing-deduping/src/main/java/io/airbyte/integrations/base/destination/typing_deduping/BaseDestinationV1V2Migrator.java

```diff
@@ -22,7 +23,7 @@ public void migrateIfNecessary(
                                  final SqlGenerator sqlGenerator,
                                  final DestinationHandler destinationHandler,
                                  final StreamConfig streamConfig)
-      throws TableNotMigratedException, UnexpectedSchemaException {
+      throws TableNotMigratedException, UnexpectedSchemaException, SQLException {
```

Comment:
nit:
```suggestion
      throws TableNotMigratedException, UnexpectedSchemaException, Exception {
```

I think we should keep jdbc/java.sql out of the top-level T+D interface, mostly for bigquery's sake